src/feinstaub-api.js

```diff
@@ -3,6 +3,8 @@
 // const URL = 'https://api.airquality.codeforafrica.org/v2/sensors/?type=PPD42NS,HPM,PMS1003,PMS3003,PMS5003,PMS6003,PMS7003,SDS021,SDS011'
 const URL = "https://api.sensors.africa/v2/nodes/";
 
+//const URL = "http://staging.api.sensors.africa/v2/nodes/";
```

Comment:
Would you be able to set this change to be an environment variable.
Then you can have 
```
.env with "https://api.sensors.africa/v2/nodes/"
``` 
and 
```
.env.development with "https://staging.api.sensors.africa/v2/nodes/"
```

Its not a must to be done in this PR

Reply:
@karimkawambwa I'll have to create a separate PR for this fixes